sys-apps/diffutils/files/loong-fix-build.patch

```diff
@@ -1,14 +1,3 @@
-From 34add045fd2ec3f1031dee961c4d9cc4285486a7 Mon Sep 17 00:00:00 2001
```

Comment:
I'd keep the commit header actually, just run `scrub-patch` from iwdevtools on it, and then include a link to where it came from.

Just add a note you included generated files. It's useful for future notes.

Reply:
Fixed. (There may not be an actual repo; apparently @sunhaiyong1978 exported the patch once and used it for all places requiring this tweak. I just directly imported from the CLFS repo when doing my porting. I added link to loongson-overlay nevertheless.)

Reply:
Hmm, might be better to link to CLFS instead.